terraso_backend/apps/graphql/schema/groups.py

```diff
@@ -82,6 +84,21 @@ class Meta:
         interfaces = (relay.Node,)
         connection_class = TerrasoConnection
 
+    def resolve_account_membership(self, info):
+        user = info.context.user
+        if user.is_anonymous:
+            return None
+        if hasattr(self, "account_memberships"):
+            if len(self.account_memberships) > 0:
+                return self.account_memberships[0]
+            return None
+        return self.memberships.filter(user=user, deleted_at__isnull=True).first()
```

Comment:
issue: if checking group memberships, we want to only include approved memberships. See https://github.com/techmatters/terraso-backend/blob/84484ea32630f6ff3fb9250c8db619e34b0fb0af/terraso_backend/apps/core/models/groups.py#L153.

Reply:
@paulschreiber I checked this, it is not needed this will be handled automatically

Reply:
for `self.memberships.filter(user=user)`, do we need to only show approved memberships?